examples/Demo/FluentUI.Demo.Client/Documentation/Components/DateTime/TimePicker/FluentTimePicker.md

```diff
@@ -0,0 +1,69 @@
+---
+title: TimePicker
+order: 0003
+route: /DateTime/TimePicker
+---
+
+# TimePicker
+
+**FluentTimePicker** offers a control that's optimized for selecting a time from a
+drop-down list or using free-form input to select a predefined time.
+If the user write an invalid time, the input field content will be deleted when the input field loses focus.
```

Comment:
If the user inputs an invalid time, the field's content will be deleted when it losses focus.